app/src/main/java/io/appium/uiautomator2/model/UiObject2Element.java

```diff
@@ -56,7 +57,36 @@ public String getStringAttribute(final String attr) throws UiObjectNotFoundExcep
         } else if (attr.equalsIgnoreCase("resourceId") || attr.equalsIgnoreCase("resource-id")) {
             res = element.getResourceName();
         } else {
-            throw new NoSuchElementAttributeException("The attribute with name '" + attr + "' was not found.");
+            throw new NoAttributeFoundException(attr);
+        }
+        return res;
+    }
+
+    public boolean getBoolAttribute(final String attr)
+            throws UiObjectNotFoundException, NoAttributeFoundException, UiAutomator2Exception {
+        boolean res;
+        if (attr.equals("enabled")) {
```

Comment:
Same as above, can we have "enabled".equals(attr) and where ever equals is called.